main.css

```diff
@@ -1 +1,59 @@
+*, ::*before, *::after {
+box-sizing: border-box;
+font-family: Georgia, sans-serif;
+font-weight: normal;
+}
 
+body {
+  padding: 0;
+  margin: 0;
+  background: linear-gradient(to right,#00AAFF, #00AA);
+}
+.calculator-grid {
+display: grid;
+justify-content: center;
+align-content: center;
+min-height: 100vh;
+grid-template-columns: repeat(4, 100px);
+grid-template-rows: minmax(120px, auto) repeat(5, 100px);
+}
+
+.calculator-grid > button {
+cursor: pointer;
+font-size: 2rem;
+border: 1px, solid white;
+outline: none;
+background-color:rgb(132, 67, 194, .75);
+
+}
+
+calculator-grid: > button:hover {
+background-color:rgba(132, 67, 194, .9);
+
+}
+
+.span-two {
+grid-column: span 2;
+
+}
+
+.output {
+grid-column: 1/ -1;
+background-color:black(0, 0, 0, .75);
+display: flex;
+align-items: flex-end;
+justify-content: space-around;
+flex-direction: column;
+padding: 10px;
+word-wrap: break-word;
+word-break: break-all;
+}
+
+.output .previous-operand {
+color: azure (240, 255, 255, 255, .75);
+font-size: 1.5rem;
+}
+
+.output .current-operand {
+color: white;
+font-size: 2.5rem;
```

Comment:
I love the start on this Shareese! Your drive is amazing. I'm continually excited to see how you grow with DOM manipulation in JavaScript. Keep on pushing and trusting yourself in the process! You got this!